modules/qr-code/backend/modules/qr-code/README.md

```diff
@@ -0,0 +1,11 @@
+# QRCode
+
+Following dependency is used to generate Qr at the backend 
+```
+pip install qrcode
+```
```

Comment:
Please correct me if I am wrong: modelbuilder does not take into account other Pipfile.locks, right?